swarm/storage/mru/topic.go

```diff
@@ -0,0 +1,56 @@
+package mru
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/ethereum/go-ethereum/common/bitutil"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/swarm/storage"
+)
+
+const topicLength = storage.KeyLength
+
+type Topic struct {
+	content [topicLength]byte
+}
+
+func NewTopic(name string, relatedContent storage.Address) Topic {
```

Comment:
Is it correct to merely truncate the name?

Reply:
Yes, there is a fixed 32 bytes for the name or `relatedContent`

Reply:
So we shouldn't return an error when the input is too long?

Reply:
We could, but rather leave it out for the sake of simplicity when using topics. I'll document it as "your string will be truncated down to 32 characters" or something.

Reply:
Ok that's fine. Please add the doc.

Reply:
Added